README.md

```diff
@@ -2,17 +2,67 @@
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
 [![All Contributors](https://img.shields.io/badge/all_contributors-1-orange.svg?style=flat-square)](#contributors-)
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
-TODO: Give a short introduction of your project. Let this section explain the objectives or the motivation behind this project. 
+BackButtonManager allows you to customize how your app reacts to the back button being pressed.
 
 # Getting Started
-TODO: Guide users through getting your code up and running on their own system. In this section you can talk about:
-1.	Installation process
-2.	Software dependencies
-3.	Latest releases
-4.	API references
-
-# Build and Test
-TODO: Describe and show how to build your code and run the tests. 
+
+## Uno projects
+
+BackButtonManager is especially well-integrated with [Uno](https://platform.uno/). Here is how to use it in a project which includes the Uno platform:
+
+* Add to your project the `Chinook.BackButtonManager.Uno` nuget package and its dependencies
+* Create a single instance of a `BackButtonManager` which you will use throughout your project.
+
+```
+var manager = new BackButtonManager();
+```
+
+* In your app's Startup, add the source which BackButtonManager uses to detect back button presses:
+
+```
+// This must be executed on the dispatcher
+var source = new SystemNavigationBackButtonSource();
+backButtonManager.AddSource(source);
+```
+
+* Add handlers for each action you want to take when the back button is pressed:
+
+```
+manager.AddHandler(new BackButtonHandler(
+						name: "TODO handler name",
```

Comment:
indentation looks off

Reply:
It's just two spaces. Weird how it looks on github.